### ISSUE TITLE
[minor] Google Play Store version not compatible with Android 14 and/or Google Pixel 6a

### ISSUE THREAD

**Issue by Zehelein** [EVNotify/EVNotify#307]
# Description

The Android app cannot be installed from the Google Play Store on Android 14 (Google Pixel 6a).

Installing the apk from the Github releases directly works perfectly fine. It shows a warning message though that it is outdated.

**Comment by SirRahikkala** [comment]
Yeah I noticed it too. Sadly it seems that the app is now abandoned by the looks of issue tracker and development changes.
